plugins/system/updatenotification/updatenotification.php

```diff
@@ -242,48 +243,32 @@ public function onAfterRender()
 			$jLanguage->load('plg_system_updatenotification', JPATH_ADMINISTRATOR, $forcedLanguage, true, false);
 		}
 
-		// Set up the email subject and body
-
-		$email_subject = Text::_('PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_SUBJECT');
-		$email_body    = Text::_('PLG_SYSTEM_UPDATENOTIFICATION_EMAIL_BODY');
-
 		// Replace merge codes with their values
 		$newVersion = $update->version;
 
 		$jVersion       = new Version;
 		$currentVersion = $jVersion->getShortVersion();
 
 		$sitename = $this->app->get('sitename');
-		$mailFrom = $this->app->get('mailfrom');
-		$fromName = $this->app->get('fromname');
 
 		$substitutions = [
-			'[NEWVERSION]'  => $newVersion,
-			'[CURVERSION]'  => $currentVersion,
-			'[SITENAME]'    => $sitename,
-			'[URL]'         => Uri::base(),
-			'[LINK]'        => $uri->toString(),
-			'[RELEASENEWS]' => 'https://www.joomla.org/announcements/release-news/',
-			'\\n'           => "\n",
+			'newversion'  => $newVersion,
+			'curversion'  => $currentVersion,
+			'sitename'    => $sitename,
+			'url'         => Uri::base(),
+			'link'        => $uri->toString(),
+			'releasenews' => 'https://www.joomla.org/announcements/release-news/'
 		];
 
-		foreach ($substitutions as $k => $v)
-		{
-			$email_subject = str_replace($k, $v, $email_subject);
-			$email_body    = str_replace($k, $v, $email_body);
-		}
-
 		// Send the emails to the Super Users
 		foreach ($superUsers as $superUser)
 		{
 			try
 			{
-				$mailer = Factory::getMailer();
-				$mailer->setSender([$mailFrom, $fromName]);
+				$mailer = new MailTemplate('joomla.updatenotification', $jLanguage->getTag());
```

Comment:
```suggestion
				$mailer = new MailTemplate('plg_system_updatenotification.mail', $jLanguage->getTag());
```
@Hackwar Can it be that @toivo is right in his comment here [https://github.com/joomla/joomla-cms/pull/28722#issuecomment-625792049](https://github.com/joomla/joomla-cms/pull/28722#issuecomment-625792049) and it has to be changed like suggested here? If so: @toivo Your test result would not be valid then because your "tweak" fixes a mistake in this PR.

Reply:
@toivo But if @Hackwar corrects it as suggested here, your test is still valid because you had already applied this correction.

Reply:
@richard67 ok, let me know if I need to test again, I would be happy to do that.